pymathics/natlang/main.py

```diff
@@ -701,10 +724,7 @@ def eval_pair(self, text1, i1, text2, i2, evaluation: Evaluation, options: dict)
             else:
                 doc2 = self._nlp(text2.value, evaluation, options)
             if doc2:
-                if (
-                    i1.get_head_name() == "System`List"
-                    and i2.get_head_name() == "System`List"
-                ):
+                if i1.get_head() == SymbolList and i2.get_head() == SymbolList:
```

Comment:
Can't `is` be used instead of `==` here?

Reply:
This and a number of other places have been corrected.